test/SecTester.Scan.Tests/CI/CiServerTests.cs

```diff
@@ -0,0 +1,100 @@
+namespace SecTester.Scan.Tests.CI;
+
+public class CiServerTests
+{
+  public static IEnumerable<object[]> ConstructorInvalidInput = new List<object[]>
+  {
+    new object[]
+    {
+      null!, ""
+    },
+    new object[]
+    {
+      string.Empty, ""
+    }
+  };
+
+  public static IEnumerable<object[]> EqualityInput = new List<object[]>
```

Comment:
OMG... Never use functions to generate test data in suites, such tests are not possible to maintain at all. I had already rewritten the whole scan tests because of this, changing just a line of code